src/components/VueDatePicker.vue

```diff
@@ -196,8 +196,13 @@ export default {
       }
       return formattedDate
     },
+    getDefaultStart () {
+      let start = new Date()
+      start.setHours(0, 0, 0, 0)
+      return start
```

Comment:
Why not like that?
```
return new Date().setHours(0, 0, 0, 0)
```

Reply:
Because new Date() is really weird. I originally tried this, and new Date().setHours(0, 0, 0, 0) returns the time in the year in milliseconds(I think), and we need the new date object.

Reply:
I see. Didn't know that!